### PR TITLE
Clean up for first release

### PR DESCRIPTION
**Pull request by iMicknl**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,7 +14,44 @@ This package is written for the Home Assistant [ha-tahoma](https://github.com/iM
...
+USERNAME = ""
+PASSWORD = ""
+
+async def main() -> None:
+    async with TahomaClient(USERNAME, PASSWORD) as client:
+        try:
+            await client.login()
+        except Exception as exception:  # pylint: disable=broad-except
+            print(exception)
+            return await client.close()
```
Comment (tetienne) [inline README.md:37]: You can  just do `return`, the `with` block will close the session automatically.

**Comment by tetienne** [comment]
Once the 0.2 published, and the ha-tahoma integration updated to target it, I suggest to rename the package name from tahoma_api to just tahoma. See https://www.python.org/dev/peps/pep-0008/#package-and-module-names 

So 

```python
from tahoma_api.client import TahomaClient
from tahoma_api.exceptions import BadCredentialsException, TooManyRequestsException
``` 

would become

```python
from tahoma.client import TahomaClient
from tahoma.exceptions import BadCredentialsException, TooManyRequestsException
```

**Comment by tetienne** [comment]
For the name of the project on pypi, _tahoma-api_ must be enough, we are on pypi, for sure it's a python project.

File: README.md [hunk 2]
```diff
@@ -14,7 +14,44 @@ This package is written for the Home Assistant [ha-tahoma](https://github.com/iM
 ## Installation
 
 ```bash
-pip install git+https://github.com/iMicknl/python-tahoma-api.git@master#tahoma_api
+pip install python-tahoma-api
```
Comment (tetienne) [inline README.md:17]: `pip install tahoma-api

File: pyproject.toml [hunk 3]
```diff
@@ -1,7 +1,7 @@
 [tool.poetry]
 name = "python-tahoma-api"
```
Comment (tetienne) [inline pyproject.toml:2]: `tahoma-api

File: README.md (hunk 2 above)
Comment (iMicknl) [reply README.md:17]: @tetienne `tahoma-api` is already taken :(

Comment (tetienne) [reply README.md:17]: Ah yes indeed, what about *pyhoma*, as suggested by @vlebourl ?

File: pyproject.toml [hunk 4]
```diff
@@ -1,20 +1,20 @@
...
+description = "Async Python wrapper to interact with internal Somfy TaHoma API"
 authors = ["Mick Vleeshouwer", "Vincent Le Bourlot", "Thibaut Etienne"]
 license = "MIT"
 readme = "README.md"
 homepage = "https://github.com/iMicknl/python-tahoma-api"
 repository = "https://github.com/iMicknl/python-tahoma-api"
 packages = [
-    { include = "tahoma_api" },
-    { include = "tahoma_api/**/*.py" },
+    { include = "tahoma" },
```
Comment (tetienne) [inline pyproject.toml:11]: We need this also? The next line is not enough?

Comment (tetienne) [reply pyproject.toml:11]: By the way, should we rename this package to pyhoma now or for the next release?

Comment (iMicknl) [reply pyproject.toml:11]: @tetienne I guess even the top one would be enough. I copied it from here: https://python-poetry.org/docs/pyproject/#packages. But I missed that they used `my_package` and `other_package`.

Comment (iMicknl) [reply pyproject.toml:11]: @tetienne I would prefer to change everything directly, before the first publish. We just need to hotfix the other component, but wouldn't be a big hassle.

Comment (tetienne) [reply pyproject.toml:11]: I think you can even remove the packages fields. Look at https://github.com/tetienne/somfy-open-api/blob/master/pyproject.toml. Don't use it and my package is ok: https://pypi.org/project/pymfy/#files

Comment (tetienne) [reply pyproject.toml:11]: OK, I let do it.

File: README.md [hunk 5]
```diff
@@ -14,7 +14,44 @@ This package is written for the Home Assistant [ha-tahoma](https://github.com/iM
...
+pip install pyhoma
+```
+
+## Getting started
+
+```python
+import asyncio
+import time
+
+from tahoma.client import TahomaClient
```
Comment (tetienne) [inline README.md:26]: Just change this import to pyhoma, and I'm good.

Comment (iMicknl) [reply README.md]: Forgot to push again..

**Pull request merged**